extract_verbs.sh

```diff
@@ -3,10 +3,67 @@
 set -e
 
 cd ..
-# Download latest Italian Wikipedia dump
-wget http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2
+# Download latest Wikipedia dump in language of Chioce
+echo "Enter wikipedia dump language e.g. en OR english"
+read LANGCODE
+#convert lnaguge code to lower case
+LANGCODE="$(echo $LANGCODE | tr '[A-Z]' '[a-z]')"
+
+#looping to select language
```

Comment:
I think you can do it like this:
```bash
case "$LANGCODE" in
    "en" | "english" )
        ...
esac
```
See example 4 [here](http://www.thegeekstuff.com/2010/07/bash-case-statement/).
Please test it and go ahead.